### PR TITLE
#164798901 Setup unit testing

### PR DESCRIPTION
**Pull request by chukwudidu**
#### Description
Setup unit testing in the project to ensure all future features are tested.
#### Type of change
- Integration of unit testing.
#### Checklist:
- [x]  Add babel to project.
- [x]  Add unit testing to the project.
#### PIVOTAL BOARD
[#164798901](https://www.pivotaltracker.com/story/show/164798901)

### CODE REVIEW

**Comment by chukwudidu** [comment]
I believe the errors flagged by HoundCi are caused by a missing ESLint configuration file which is yet to be merged into the *staging* branch

**Comment by TheDrizzyWay** [comment]
Looks good. Build is now passing.

**Comment by geekdanitek** [comment]
Good job.

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,5 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+# Nyc test output
+.nyc_output
```
Comment (paradisekelechi) [inline .gitignore:38]: @cupidcoder Is this part of setting up unit testing?

Comment (chukwudidu) [reply .gitignore:38]: @paradisekelechi not directly but after I ran tests, it generated the .nyc_output folder which I didn't want to be pushed to the online repo

**Pull request merged**